osu.Game/Screens/Play/Player.cs

```diff
@@ -407,7 +413,10 @@ private void performUserRequestedExit()
             if (canPause)
                 Pause();
             else
+            {
+                samplePause?.Stop();
```

Comment:
Going into the pause screen and then pressing Alt+F4 to exit back to the menu will keep the sample playing in the background. You'd likely want this somewhere inside `OnExiting()` instead.